tests/test_basic_auth.py

```diff
@@ -0,0 +1,143 @@
+import os
+import socket
+from urllib import parse as url_parse
+
+import requests
+import pytest
+import requests_mock
+
+from domino import Domino, bearer_auth
+
+
+def domino_is_reachable(url=os.getenv('DOMINO_HOST_KEY_NAME'), port='443'):
+    """
+    Confirm that a deployment is acessible for tests that require it.
+    """
+    if url is None:
+        return False
+
+    try:
+        fqdn = url_parse.urlsplit(url).netloc
+        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        s.connect((fqdn, int(port)))
+        is_reachable = True
+    except socket.gaierror:
+        print(f"{fqdn}:{port} is not reachable")
+        is_reachable = False
+    finally:
+        s.close()
+
+    return is_reachable
+
+
+@pytest.fixture
+def mock_domino_version_response():
+    """
+    Simulate a valid repsonse to an authenticated /version GET request.
+    """
+    version_info = {
+        "buildId": "12345",
+        "buildUrl": "https://some.server.com/domino/12345",
+        "commitId": "123deadbeef456",
+        "commitUrl": "https://some.repo.com/domino/commit/123deadbeef456",
+        "timestamp": "2021-06-14T16:50:02Z",
+        "version": "9.9.9"
+    }
+
+    with requests_mock.Mocker() as mock_endpoint:
+        mock_endpoint.get("http://mock.domino.com/version", json=version_info, status_code=200)
```

Comment:
Changed to use our real domain.
```suggestion
        mock_endpoint.get("http://mock.dominodatalab.com/version", json=version_info, status_code=200)
```

Reply:
But it's not a dummy domain: `domino.com` is someone else's domain. So is `somecompany.com`. Both of these are registered domains. I know you're not making real calls, but since this is OSS, nor do I want bs crawlers to see that and associate that domain with our code. Consider it just wanting not to have weird associations on the Internet.

Reply:
Oh, and I missed the `repo.com` usage. I don't know why the `domino.com` stuck out: maybe another reason is it's metasimilar but not our correct domain name?